lib/t/matches.ex

```diff
@@ -265,6 +265,10 @@ defmodule T.Matches do
     |> Repo.one()
   end
 
+  def match_after_live_call(user_ids) do
+    [user_id_1, user_id_2] = Enum.sort(user_ids)
```

Comment:
@akhrail1996 what side-effects should there be, notifications, etc?

Reply:
notifications are not needed, maybe just a broadcast so that LIVE Feed updates its UI (turning mate from regular profile into a match) - but I would leave that for later

as for Normal Feed no side-effects are needed, since both users will join Normal Feed freshly (therefore they will be rendered as matches correctly already at this point)

Reply:
moved to https://github.com/getsince/test3/issues/476